### PR TITLE
Add Traditional Chinese README.md

### PR DESCRIPTION
**Pull request by ascodeasice**

### CODE REVIEW

**Review by TenTraicion — CHANGES_REQUESTED** [review]
@ascodeasice, looks fine to me. But i will recommend some changes:
1. Change the file name to: `CHINESE.trad.md` || `CHINESE.traditional.md` (i prefer the shortest one though)
2. Input in the language table alphabetically. (Optional)

Thanks for all the hard work. and we are looking forward to more contributions of your.

-Regards

**Comment by ascodeasice** [comment]
Hi, I used Chinese (Traditional) in table since there's sorting

File: README.md [hunk 1]
```diff
@@ -82,11 +82,11 @@ You will make a copy of the card template inside the HTML file and customize it
...
 | :---------------------------------------------: | :---------------------------------------: | :-----------------------------------------: | :---------------------------------------: | :---------------------------------------: |
-|      [Hindi](/translations/README/HINDI.md)      | [Italian](/translations/README/ITALIAN.md) | [Japanese](/translations/README/JAPANESE.md) |  [Korean](/translations/README/KOREAN.md)  |  [Polish](/translations/README/POLISH.md)  |
+|  [German](/translations/README/German.md)  |      [Hindi](/translations/README/HINDI.md)      | [Italian](/translations/README/ITALIAN.md) | [Japanese](/translations/README/JAPANESE.md) |  [Korean](/translations/README/KOREAN.md)  |  [Polish](/translations/README/POLISH.md)  |
 | [Portuguese](/translations/README/PORTUGUESE.md) | [Russian](/translations/README/RUSSIAN.md) |  [Serbian](/translations/README/SERBIAN.md)  | [Spanish](/translations/README/SPANISH.md) | [Turkish](/translations/README/TURKISH.md) |
```
Comment (TenTraicion) [inline README.md:88]: Polish is missing

File: translations/translation.md [hunk 2]
```diff
@@ -18,9 +18,9 @@ All Translations regarding Project Documentations are welcome. There are certain
 
 ## README Translations
 
-| [Arabic](README/ARABIC.md) | [Bangla](README/BANGLA.md) | [English](../README.md) | [French](README/FRENCH.md) | [German](README/German.md) |
+| [Arabic](README/ARABIC.md) | [Bangla](README/BANGLA.md) | [Chinese (Traditional)](README/CHINESE_TRADITIONAL.md) | [English](../README.md) | [French](README/FRENCH.md)
 | :------------------------: | :------------------------: | :---------------------: | :------------------------: | :------------------------: |
-| [Hindi](README/HINDI.md)   | [Italian](README/ITALIAN.md) | [Japanese](README/JAPANESE.md) | [Korean](README/KOREAN.md)   | [Polish](README/POLISH.md)   |
+ [German](README/German.md) | [Hindi](README/HINDI.md)   | [Italian](README/ITALIAN.md) | [Japanese](README/JAPANESE.md) | [Korean](README/KOREAN.md)   | [Polish](README/POLISH.md)   |
 | [Portuguese](README/PORTUGUESE.md) | [Russian](README/RUSSIAN.md) | [Serbian](README/SERBIAN.md)   | [Spanish](README/SPANISH.md) | [Turkish](README/TURKISH.md) |
```
Comment (TenTraicion) [inline translations/translation.md:24]: Polish is missing

File: translations/translation.md [hunk 3]
```diff
@@ -18,9 +18,9 @@ All Translations regarding Project Documentations are welcome. There are certain
...
 | [Ukrainian](README/UKRAINIAN.md) |
```
Comment (TenTraicion) [inline translations/translation.md:25]:
```suggestion
| [German](README/German.md) | [Hindi](README/HINDI.md)   | [Italian](README/ITALIAN.md) | [Japanese](README/JAPANESE.md) | [Korean](README/KOREAN.md)   | 
| [Polish](README/POLISH.md) | [Portuguese](README/PORTUGUESE.md) | [Russian](README/RUSSIAN.md) | [Serbian](README/SERBIAN.md)   | [Spanish](README/SPANISH.md) |
| [Turkish](README/TURKISH.md) | [Ukrainian](README/UKRAINIAN.md) |
```

**Review by TenTraicion — CHANGES_REQUESTED** [review]
@ascodeasice, need some more changes, mentioned below:

**Comment by ascodeasice** [comment]
Done, thanks for reminding!

**Review by TenTraicion — APPROVED** [review]

**Comment by TenTraicion** [comment]
@ascodeasice, Congratulation on your PR Merge, and Thanks for your Contribution. Don't forget to add your card in our project. Looking forward to your future contributions.

Welcome to Open Source Community. You can follow the [Next Steps](https://github.com/Syknapse/Contribute-To-This-Project#next-steps) now. You are also welcome to join our [Discord Server: https://discord.gg/tWkvS4ueVF](https://discord.gg/tWkvS4ueVF) and be a part of our community.

**Pull request merged**